.github/workflows/firebase-hosting-pull-request.yml

```diff
@@ -11,7 +11,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: subosito/flutter-action@v2
         with:
-          channel: 'stable'
```

Comment:
You don't have to remove the `channel` here, although I'm not sure it makes any difference whether it's there or not tbf.

Reply:
Took a look at some examples and seems like it's better practice to include the channel, so I've added it back.